relay-server/src/services/health_check.rs

```diff
@@ -225,6 +226,8 @@ impl Service for HealthCheckService {
                 });
             }
         });
+
+        j1 // TODO: should return j1 + j2
```

Comment:
We have a few places where the spawn handler spawns more than one task. In a follow-up, we should transform these to something like

```rust
tokio::spawn(async {
    let subtask = tokio::spawn(async {...});
    /// ...
    subtask.await;
});